src/dotnet/OpenAPI.Validator/Model/Utilities/ValidationUtilities.cs

```diff
@@ -23,6 +23,36 @@ public static class ValidationUtilities
         public static readonly Regex ListBySidRegEx = new Regex(@".+_(List|ListBySubscriptionId|ListBySubscription|ListBySubscriptions)$", RegexOptions.IgnoreCase);
         public static readonly Regex ListByRgRegEx = new Regex(@".+_ListByResourceGroup$", RegexOptions.IgnoreCase);
 
+        public static IEnumerable<string> GetTenantResourceModels(IEnumerable<string> resourceModels, ServiceDefinition serviceDefinition)
+        {
+            List<string> tenantResourceModels = new List<string>();
+            if(serviceDefinition.Paths.Any())
+            {
+                foreach (KeyValuePair<string, Dictionary<string, Operation>> path in serviceDefinition.Paths)
```

Comment:
@ravbhatnagar could you check on this logic, to make sure the lookup of tenant level resource is appropriate?